src/cli.js

```diff
@@ -156,3 +169,38 @@ async function upgradeAndDockerize (name) {
   await autoUpgrade();
   dockerize(name);
 }
+
+function updateLastVersionUsed() {
+  const fileHeader = "DO NOT MODIFY";
+
+  // Check version in .gluestick file
+  const gluestickDotFile = path.join(process.cwd(), ".gluestick");
+  var fileContents = fs.readFileSync(gluestickDotFile, {encoding: "utf8"}).replace(fileHeader, "");
+  // If the dot file only had the header (and possibly a new line character), then it definitely doesn't have the version check and is old
+  fileContents = (fileContents.length <= 1)? '{"version": "`unknown version`"}': fileContents;
+  var json = JSON.parse(fileContents);
+  if (shouldUpdateForProjectGluestickVersion(json.version)) {
+    console.log(chalk.yellow("This project is configured to work with versions >= " + json.version + " Please upgrade your global `gluestick` module with `sudo npm install gluestick -g"));
+  }
+
+  // update version in dot file
+  var newContents = fileHeader + "\n";
+  newContents += JSON.stringify({version: getVersion()});
+  fs.writeFileSync(gluestickDotFile, newContents);
+} 
+
+
+
+function shouldUpdateForProjectGluestickVersion(projectGluestickVersion) {
+  var mine = getVersion().split('.');
+  var project = projectGluestickVersion.split('.');
+
+  if (mine[0] !== project[0] 
```

Comment:
Is there a reason for splitting on the "." instead of just comparing `getVersion() === projectGluestickVersion` ?